docs/explanation/tracing.md

```diff
@@ -1,5 +1,47 @@
 (tracing)=
 # Tracing
+(tracing-users)=
+## Tracing users
+
+Instrument generously, so that more context is captured than is immediately needed; this ensures that enough data is available for each role:
+
+**Charm developers** get more visibility during local development: how Juju hooks and charm library processing is connected to the key code paths in their charms.
+
+**Field engineers** can inspect the trace data from live deployments, ideally avoiding the need to reproduce failures in staging.
+
+**SREs** may want to set latency and error rate alerts on specific elements of trace data preemptively. They can also get more insights to help root-cause analysis across services.
+
+**QA teams** can incorporate trace assertions into integration tests to simplify interoperability testing.
+
+At the same time, we do not recommend deriving **business metrics** from charm trace data, as the workload trace data should be the primary source.
+
+## Relations
+
+The recommended declaration is:
+
+```yaml
+requires:
+  charm-tracing:
+    interface: tracing
+    limit: 1
+    optional: true
+  receive-ca-cert:
+    interface: certificate_transfer
+    limit: 1
+    optional: true
+```
+
+The traced charm must be the requirer in the relation. Trace data sinks, like Tempo, are the providers.
+
+The `limit` attribute is validated at charm initialisation time, because the tracing infrastructure can only handle one destination at a time. Thus it doesn't make sense for the traced app to be related to multiple providers simultaneously.
+
+The `optional` attribute is recommended, as it gives a hint that this charm can be deployed with or without tracing.
```

Comment:
I'd rather we didn't say that this is recommended. We expect charms to provide this field now, so we shouldn't be suggesting that they could choose not to. I feel like "optional: true" is pretty self-explanatory, but if we do want to explain it, then we can just say:

```suggestion
The `optional` attribute informs users that this charm can be deployed with or without tracing.
```